docs/tutorials/dev_add_test_group.md

```diff
@@ -0,0 +1,2466 @@
+    def validate(self):
+        """
+        Compare ``temperature``, ``salinity``, and ``layerThickness`` in the 
+        ``initial_state`` step with a baseline if one was provided.
+        """
+        super().validate()
+
+        variables = ['temperature', 'salinity', 'layerThickness']
+        compare_variables(test_case=self, variables=variables,
+                          filename1='initial_state/initial_state.nc')
+```
+We check salinity, temperature and layer thickness in the initial state step.
+Since we only provide `filename1` in the call to 
+{py:func}`polaris.validate.compare_variables()`, we will only do this 
+validation if a user has set up the test case with a baseline, see 
+{ref}`dev-validation`.
+
+(dev-tutorial-add-test-group-testing-a-step)=
+
+### Test things out!
+
+It's a good idea to test things out after adding each step to a test case.
+Before we add any more steps or test cases, we'll run `default` and make sure 
+we can create the initial condition.  It would be good to make sure what we've 
+done so far works well before we move on.  
+
+The first way to test things out is just to list the tests and make sure your
+new ones show up:
+
+```bash
+$ polaris list
+
+Testcases:
+   0: ocean/baroclinic_channel/10km/default
+   1: ocean/baroclinic_channel/10km/decomp_test
+   2: ocean/baroclinic_channel/10km/restart_test
+   3: ocean/baroclinic_channel/10km/threads_test
+   4: ocean/baroclinic_channel/1km/rpe_test
+   5: ocean/baroclinic_channel/4km/rpe_test
+   6: ocean/baroclinic_channel/10km/rpe_test
+   7: ocean/global_convergence/qu/cosine_bell
+   8: ocean/global_convergence/icos/cosine_bell
+   9: ocean/yet_another_channel/1km/default
+   10: ocean/yet_another_channel/4km/default
+   11: ocean/yet_another_channel/10km/default
+```
+
+If they don't show up, you probably missed a step (adding the test group to the
+component or the test case to the test group).  If you get import errors or
+syntax errors, you'll need to fix those first.
+
+If listing works out, it's time to set up one of your tests.  Probably start
+with one that's pretty light weight and fast to run.  In this case, that's the
+10 km `defatult` (test number 11):
+
+```bash
+$ polaris setup -n 11 -p <E3SM_component> -w <work_dir>
+```
+See {ref}`dev-polaris-setup` for the details.  If that works, you're ready to
+do a test run.  If you get errors during setup, you have some debugging to do.
+
+You can run the test with a job script or an interactive node.  For debugging,
+the interactive node is usually more efficient.  To run the test case, open a 
+new terminal, go to the work directory, start an interactive session on
+however many nodes you need (most often 1 when you're just debugging something
+small) and for a long enough time that your debugging doesn't get interrupted,
+e.g. on Chrysalis:
+```bash
+$ cd <work_dir>
+$ srun -N 1 -t 2:00:00 --pty bash
+```
+Then, on the interactive node, source the local link the load script and run:
```

Comment:
```suggestion

Let's navigate into the test case directory and see what it looks like:
```bash
$ cd ocean/yet_another_channel/10km/default
$ ls
default.cfg  load_polaris_env.sh   initial_state  job_script.sh      test_case.pickle
[```end bash]

If we open up `default.cfg` we can see that it contains our newly added `yet_another_channel` section along with a bunch of other sections. Let's go to the `test_case` section, where you will see `steps_to_run = initial_state`. This means that when we run our case, the initial condition should be generated if we didn't make any mistakes in setting up the step (fingers crossed!).

Then, on the interactive node, source the local link the load script and run:
```

Reply:
I believe you can use quadruple-backticks and nest triple-backticks inside of them (and I can use quintuple backticks to demonstrate quadruple-backtick).
`````
````suggestion
Then, on the interactive node, source the local link the load script and run:
```bash
blah
```
````
`````

Reply:
Thanks! I was wondering how to make that work